src/frontend/typer/vedtak/vedtak.ts

```diff
@@ -28,6 +30,8 @@ export const årsakAvslagTilTekst: Record<ÅrsakAvslag, string> = {
     MANGELFULL_DOKUMENTASJON: 'Mangelfull dokumentasjon',
     HAR_IKKE_UTGIFTER: 'Har ikke utgifter',
     RETT_TIL_UTSTYRSSTIPEND: 'Rett til utstyrsstipend',
+    HAR_IKKE_MER_UTGIFTER: 'Har ikke nødvendige merutgifter',
```

Comment:
```suggestion
    HAR_IKKE_MERUTGIFTER: 'Har ikke nødvendige merutgifter',
```
Det går an å skrive "mer utgifter", men det betyr ikke helt det samme som ordet "merutgifter" 😌

https://ordbokene.no/nno/bm,nn/merutgift

Reply:
Fikset